api.planx.uk/modules/analytics/metabase/collection/createCollection.ts

```diff
@@ -0,0 +1,20 @@
+import { createMetabaseClient } from "../shared/client.js";
+import type { NewCollectionParams } from "./types.js";
+
+const client = createMetabaseClient();
+
+export async function createCollection(
+  params: NewCollectionParams,
+): Promise<any> {
+  const transformedParams = {
+    name: params.name,
+    parent_id: params.parentId,
+  };
+
+  const response = await client.post(`/api/collection/`, transformedParams);
+
+  console.log(
```

Comment:
Yep great question! Generally we only leave this in if we're debugging or want to get more visibility on a feature which is raising errors. If we leave them in as just status updates it can lead to very noisy logs and tests.